### DISCUSSION TITLE
multiple reminders per day

### DISCUSSION THREAD

**Discussion by KoljaL** [iSoron/uhabits#824] [Feature Requests]
Great App, but is there a way to set more then one reminder per day?
Maybe with a random remindertime, so i just have to set: do situps 3times every day, but not on sunday

**Comment by hiqua** [comment]
You mean that you'd like to get a reminder back once you've swiped it away?

**Comment by KoljaL** [comment]
Yes, the task is then not done for the day, but only for the next xx hours.
And the number of hours could randomly be set between 1 and 23.
If you think further, you should also take into account that the app doesn't remind me of my workout at 2am :-)

**Comment by hiqua** [comment]
There is already a "Later" option in the notification: https://github.com/iSoron/uhabits/discussions/768#discussioncomment-351796

Your suggestion seems to be to allow the user to select "random time" rather than choosing it. I guess why not, but on the other hand you can just select at random yourself. I don't think users would benefit much from an additional option that would just say `random`.

**Reply by lauren-martini** [reply]
Hello! This is a really cool app. I am also interested in this feature, but not the randomness part. I could use multiple reminders for completing different increments of a measurable habit.

Let's say your goal is to exercise 60 minutes per day, then you could set 2 reminders - once in the morning, once in the evening for each 30 minute session.

Or if your goal is to drink 3 L of water, then you could set 6 reminders throughout the day, each one telling you to drink 500 ml. This example with many reminders is one case where I don't think it makes sense to split the goal into multiple habits, each with one reminder.

**Comment by leinestein** [comment]
I agree. I also miss this feature in the app.
If I want to do a habit twice a day, I would like to set a reminder twice a day.
